wic/src/main/java/edu/asu/diging/wic/web/GetConceptWikiSummaryController.java

```diff
@@ -0,0 +1,43 @@
+package edu.asu.diging.wic.web;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.client.HttpClientErrorException;
+
+import edu.asu.diging.wic.core.conceptpower.IConceptpowerCache;
+import edu.asu.diging.wic.core.model.IConcept;
+import edu.asu.diging.wic.core.wiki.IWikiConnector;
+
+@Controller
+@PropertySource(value = "classpath:/config.properties")
+public class GetConceptWikiSummaryController {
+
+    @Value("${wikipedia.url.pattern}")
+    private String wikiRegex;
+
+    @Autowired
+    private IConceptpowerCache cache;
+
+    @Autowired
+    private IWikiConnector wikiConnector;
+
+    @GetMapping("/wiki-summary/concept/{conceptId}")
```

Comment:
this should be `/concept/{conceptId}/wiki-summary`, and then the spring security change should be reverted.